.github/pull_request_template.md

```diff
@@ -14,10 +14,19 @@ and code of conduct https://github.com/grafana/k6/blob/master/CODE_OF_CONDUCT.md
 -->
 
 - [ ] I have performed a self-review of my code.
-- [ ] I have added tests for my changes.
-- [ ] I have run linter locally (`make lint`) and all checks pass.
-- [ ] I have run tests locally (`make tests`) and all tests pass.
 - [ ] I have commented on my code, particularly in hard-to-understand areas.
+- [ ] I have added tests for my changes.
+- [ ] I have run linter and tests locally (`make lint` and `make tests`) and all pass.
```

Comment:
Do we really want to modify this? 🤔 I have no strong opinion, neither for nor against.

Reply:
Me neither. But, just simplified a little 😊

Reply:
If we want to simplify we could refer to a `make check` which is a shortcut for running both of them. 

_Sidenote: it's useful to set up as a hook (I can share it with you) which automatically runs before pushing to the repo_